### ISSUE TITLE
"Alt text" is not provided for the images on the select a scenario page: A11y_ML.NET Model Builder_Non text Content.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3020]
### GitHubTags

#A11ySev2; #A11yTCS;#DesktopApp; #Win32;#Model Builder - Machine Learning for .NET Developers;#BM-MlnetModelBuilder-Win32-March25;#Non-Text Content; #Win32;#WCAG1.1.1;#A11yMAS; 

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3
### Repro Steps:
**Prerequisites:**

1. Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Verify that Alt text is not provided for the images on the select a scenario page

### Actual Result: 
Alt text is not provided for the images on the select a scenario page.
### Expected Result:
Alt text should be provided for the images and screen reader should announce it or focus should not go on the decorative images.
### User Impact:
Screen reader users will not get the information about the image as screen reader not announcing anything for it.

### Attachment:
https://github.com/user-attachments/assets/51342ace-49e2-4c18-8a7a-ad4bc7a0f3bb

**Comment by AnKushSingh05** [comment]
Still issue is not fixed
Microsoft Visual Studio Enterprise
Channel: Canary
Version: 18.3.0 Insiders [11308.173.main]
When focus lands on the image screen reader remains silent either this focus should be removed or alt text should be provided.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/feee2b2a-d649-4a41-aa19-49dc4c689bdc" />

**Comment by AnKushSingh05** [comment]
Issue is not fixed in version 17.19.2.2511501
Screen reader remains silent when focus lands on the images, this extra focus should be removed as these are decorative images
<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/6b17a4d9-9a4e-4555-830d-dea8e6fdf98c" />

**Comment by AnKushSingh05** [comment]
Issue is fixed in version 17.19.2.2536801

https://github.com/user-attachments/assets/cc0fd1d4-ff7d-4e7c-90e9-0d7b9de1c94d
